bin/Analysis.cpp

```diff
@@ -151,6 +155,9 @@ void Analysis::createHistograms() {
 			<< endl;
 	lastNumberOfHistograms = numberOfHistograms;
 
+	pseudoTopAnalyser_->createTrees();
```

Comment:
I had to do :

if ( eventReader->getSeenDatatypes()[DataType::TTJets] ) 

as the other method doesn't work.  I believe currentEvent->getDataType() will only work after  Event::setDataType has been called in initiateEvent()